### PR TITLE
Updated Grafana docs for GA release

### PR DESCRIPTION
**Pull request by hwandersman**
*Description of changes:*

* Updated README instructions for creating a workspace.
* Updated Grafana doc with instructions for AMG and additional warning for using the AssumeRole ARN in a Cloud9 EC2 environment.
* Updated the IAM policy in create_grafana_dashboard_role.py to give minimum permissions for KVS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,23 +108,37 @@ Note: These instructions have primarily been tested for Mac/Linux/WSL environmen
 
 3. Create an IoT TwinMaker workspace.
 
-   Different Digital Twin applications use different resources. Run the following command to create a role for our workspace that has the necessary permissions for this sample application. Note that you will use the role name in the output in the next step.
+   a. Create a Grafana dashboard IAM role
```
Comment (johnnyw-aws) [inline README.md:111]: Should workspace role creation still be first as it's required for workspace

Maybe the following?

a. TwinMaker execution role
b. Grafana role
c. Create workspace

Comment (hwandersman) [reply README.md]: Updated the README to reflect this

File: README.md [hunk 2]
```diff
@@ -103,32 +108,52 @@ Note: These instructions have primarily been tested for Mac/Linux/WSL environmen
 
 3. Create an IoT TwinMaker workspace.
 
-   Different Digital Twin applications use different resources. Run the following command to create a role for our workspace that has the necessary permissions for this sample application. Note that you will use the role name in the output in the next step.
+   a. Create a TwinMaker execution role
```
Comment (johnnyw-aws) [inline README.md:111]: "IoT TwinMaker" (may be other places)

Comment (hwandersman) [reply README.md]: Updated everywhere in the README and grafana_local_docker_setup.md

**Pull request merged**